src/main/java/build/buf/protovalidate/internal/evaluator/EvaluatorBuilder.java

```diff
@@ -165,11 +172,10 @@ private void buildMessage(Descriptor desc, MessageEvaluator msgEval)
       try {
         DynamicMessage defaultInstance =
             DynamicMessage.newBuilder(desc)
-                .mergeFrom(new byte[0], EXTENSION_REGISTRY)
+                .mergeFrom(new byte[0], extensionRegistry)
```

Comment:
Pretty sure this line does nothing. Remove?

Reply:
Yep. Removed.